### ISSUE TITLE
"Provider not found" only on mobile

### ISSUE THREAD

**Issue by Trynight** [nguyenphu27/pancake-swap-exchange-testnet#17]
Practically after releasing the code on any hosting, it works perfectly for me on desktop (both on mainnet and on testnet), while if I try to connect via Metamask App (Android), "Provider Not Found" comes out.

![immagine](https://user-images.githubusercontent.com/58222452/159467300-67bb199f-20d8-400b-8b69-8d3b91db7c69.png)

**Comment by nguyenphu27** [comment]
I guess your metamask app is not setting up network properly

**Comment by Trynight** [comment]
> 

No, for sure I have tested with the correct networks. Now I'm trying to get working your code with mainnet and works on desktop but not on mobile.

**Comment by nguyenphu27** [comment]
Have you tried access my website 'https://pancakeswap-testnet.netlify.app/'? The code is exactly the same in this repo.
I can access testnet in metmask android app normally.
